profile_library/signify/LCD003/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-07-24T06:28:09.271856",
+  "measure_description": "Measured with utils/measure script, 2 lights",
+  "measure_device": "Shelly 1PM",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.13.4:docker"
+  },
+  "name": "LCD003",
```

```suggestion
  "name": "Hue Akari Downlight",
```

you can use the marketed name here, this will be shown in the library UI.
https://powercalc.lauwbier.nl